view/shared/icons/README.md

```diff
@@ -0,0 +1,5 @@
+To add new icons add **.svg** file to assets directory and than run
+
+```sh
+pnpm run build
```

Comment:
Could you please use the following README file for this purpose:
view\shared\ui\README.md

the command can also be simplified as:
`pnpm build`